src/main/java/com/ait/lienzo/client/core/shape/wires/IControlPointsAcceptor.java

```diff
@@ -0,0 +1,63 @@
+package com.ait.lienzo.client.core.shape.wires;
+
+public interface IControlPointsAcceptor
+{
+    public static final IControlPointsAcceptor ALL  = new DefaultControlPointsAcceptor(true);
+
+    public static final IControlPointsAcceptor NONE = new DefaultControlPointsAcceptor(false);
+
+    public boolean add(WiresConnector connector,
+                       int index,
+                       double x,
+                       double y);
+
+    public boolean move(WiresConnector connector,
+                       int index,
+                        double tx,
+                        double ty);
+
+    public void update(WiresConnector connector);
```

Comment:
the update uses only the connector as a parameter?

Reply:
hehe good catch.. yeah! In fact it was not clear for me neither... I've changed the API for the IControlPointsAcceptor, just a single move method that takes an array Point2D's, which are the locations to apply for each control point. WDYT? So replaced both methods by a single one, also in Stunner it does the check for each CP against the candidate location, and only applies commands for the given CPs updated, if any.